test/eks-cluster-test/reset-cluster

```diff
@@ -1,32 +1,36 @@
 #!/bin/bash
 
-echo "Uninstall Helm charts ..."
-helm list -a -A |
-    tail -n +2 |
-    tr -s "\t" " " |
-    awk '{ printf("helm uninstall -n %s %s\n", $2, $1) }' |
-    bash -s
-echo "✅ done"
+function reset_cluster {
+    echo "Resetting cluster"
+    charts=$(helm ls --all --short)
+    if [[ ! -z "$charts" ]]; then
+        helm del $charts || :
+    fi
+    system_charts=$(helm ls --all --short --namespace kube-system)
+    if [[ ! -z "$system_charts" ]]; then
+        helm del $system_charts --namespace kube-system || :
+    fi
+    for node in $(kubectl get nodes | tail -n+2 | cut -d' ' -f1); do
+        kubectl uncordon $node
+        kubectl taint node $node aws-node-termination-handler/scheduled-maintenance- || true
+        kubectl taint node $node aws-node-termination-handler/spot-itn- || true
```

Comment:
To handle future changes, could this just remove all taints under `aws-node-termination-handler/`?

Reply:
I think there's value in seeing which taints are applied/removed by nth explicitly, but I see your point with respect to scaling with more taints. Since there are only 2 right now, would you be good with leaving as-is and addressing once it starts to grow? 

As a side note, this is the same logic used in the k8s-local test so once we consolidate test drivers I think we can address this logic as well